src/Microsoft.Sbom.Api/Config/Args/GenerationArgs.cs

```diff
@@ -116,6 +116,15 @@ public class GenerationArgs : GenerationAndValidationCommonArgs
     [ArgDescription("If set to true, we will attempt to fetch license information of packages detected in the SBOM from the ClearlyDefinedApi.")]
     public bool? FetchLicenseInformation { get; set; }
 
+    /// <summary>
```

Comment:
A new parameter should be documented in both `docs\sbom-tool-arguments.md` and `src\Microsoft.Sbom.Targets\README.md`

Reply:
Done in the new commit.